tests/errors/cpdef_vars.pyx

```diff
@@ -0,0 +1,18 @@
+# tag: warnings
+
+cpdef str a = "123"
+cpdef b = 2
+
+cdef class C:
+    cpdef float c
```

Comment:
Hmm... I wonder if we should actually make this "cdef public" analogous to cpdef methods. (The others clearly require a warning becoming an error until we can support them.)

Reply:
I can see the logic to that. However, my opinion would be not to add second way to do the same thing. It might be worth revisiting if/when `cpdef` module variables are implemented (assuming that syntax is the one used...) but right now it doesn't feel like a huge inconsistency not to support it.

It isn't a hugely strong opinion though.

Reply:
Fair point. I'm fine with getting this in and possibly revisiting the syntax later.

Reply:
> However, my opinion would be not to add second way to do the same thing.

I generally agree, although I've always had a problem with `cdef public attr`: the `public` modifier is otherwise used to export C symbols and here it does something entirely different. `cpdef attr` seems much more appropriate for this.

Reply:
Arguably Cython 3 is a good time to make this change, so do that instead of https://github.com/cython/cython/pull/3963